### PR TITLE
Remove duplicate SSP Authorization Boundary Desc

### PR DESCRIPTION
**Pull request by kylelaker**
This removes the duplicate field and rebalances the layout to make it
flow a little more naturally.

### CODE REVIEW

**Comment by kylelaker** [comment]
image

File: src/components/OSCALSystemCharacteristics.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default function OSCALSystemCharacteristics(props) {
             </Grid>
             <Grid
               item
-              xs={5}
+              xs={6}
```
Comment (zclarkEDC) [inline src/components/OSCALSystemCharacteristics.js:59]: https://mui.com/api/grid/

We could also do `xs={auto}` right?
Does that look better?

**Review by zclarkEDC — APPROVED** [review]

File: src/components/OSCALSystemCharacteristics.js (hunk 1 above)
Comment (kylelaker) [reply src/components/OSCALSystemCharacteristics.js:59]: Umm I guess we could? But nothing else here really does so I didn't want to get too crazy with it in a bug fix.

Comment (kylelaker) [reply src/components/OSCALSystemCharacteristics.js:59]: Might make a good `enhancement` ticket?

**Review by rgauss — APPROVED** [review]

**Pull request merged**